test/tests/data/search/a.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>hello</title>
+  </head>
+  <body>
+    <p>mjktkiuewf</p>
```

Comment:
If it's only a couple seconds, that seems fine.
A little randomness (for instance, as to what the database is actually populated) is a good thing to have for your tests, in my opinion, since it increases the possibility of a failure. :)